scripts/measures/2018/merge-measures-data.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+const path = require('path');
+
+const outputPath = process.argv[4];
+
+function mergeMeasures() {
+  const allJson = [];
+  // first command-line argument starts at position 2, last argument
+  // is the output path. everything in between is an input measures file
+  for (let i = 2; i < process.argv.length - 1; i++) {
+    const measureStagingPath = path.join(__dirname, process.argv[i]);
+    const measureJson = JSON.parse(fs.readFileSync(measureStagingPath, 'utf8'));
+    allJson.push(measureJson);
+  }
+
+  const combinedMeasures = [].concat(...allJson);
+  const combinedJson = JSON.stringify(combinedMeasures, null, 2);
+  fs.writeFileSync(path.join(__dirname, outputPath), combinedJson);
+}
+
+mergeMeasures();
```

Comment:
Should we have any tests for this function? Are there tests from 2017 that we could use for it?

Reply:
good point, yes I'll add a test here. nope, no existing tests

Reply:
added!